tests/generate_packageTest.py

```diff
@@ -0,0 +1,24 @@
+import os
```

Comment:
but they are? except for the other branch of the if cond in `touch()`. You might want to try covering both cases, file existing and not existing.